lib/utilities.bash

```diff
@@ -60,15 +60,21 @@ function _bash-it-array-dedup() {
 	printf '%s\n' "$@" | sort -u
 }
 
-# Outputs a full path of the grep found on the filesystem
+# Runs `grep` with *just* the provided arguments
 function _bash-it-grep() {
-	: "${BASH_IT_GREP:=$(type -P egrep || type -P grep)}"
-	printf "%s" "${BASH_IT_GREP:-/usr/bin/grep}"
+	: "${BASH_IT_GREP:=$(type -P grep)}"
+	"${BASH_IT_GREP:-/usr/bin/grep}" "$@"
 }
```

Comment:
I could not find any usages of the original function and not sure that, for its purpose, it was named correctly, considering it _looked_ like a sister function to `_bah-it-egrep`